devdocs/create-new-release-pipeline.md

```diff
@@ -0,0 +1,49 @@
+3. You are now on a page that says "New Pipeline". Fill in the fields as follows:
+    - Name: `julia-release-1.9`
+    - Description: `https://github.com/JuliaLang/julia`
+    - Git Repository:
+        - Step i. Click on the dropdown that says "Any account", and click on `JuliaLang`.
+        - Step ii: Click on the text field that says "git@github.com:your/repo.git", and from the dropdown menu that appears, click on `JuliaLang/julia`.
+        - Step iii: Next to "Checkout using", click on the radio button for `HTTPS`. Once you do that, the URL should automatically change to `https://github.com/JuliaLang/julia.git`
+    - Auto-create webhooks: make sure that this checkbox IS checked.
+    - Teams: select the following two teams:
+        - `[ALL USERS] Base Julia CI (build and read)`
+        - `Base Julia CI (Slack notifications)`
+4. At the bottom of the page, click on the green "Create Pipeline" button.
+5. Go to https://buildkite.com/julialang/julia-release-1-dot-9 and click on the white "Edit steps" button.
+6. Delete ALL text in the text box.
+7. Into the text box, paste the exact contents from the folllowing file: https://github.com/JuliaCI/julia-buildkite/blob/main/pipelines/main/0_webui.yml
+8. Click on the white "Save Steps" button.
+9. Go to https://buildkite.com/julialang/julia-release-1-dot-9 and click on the white "Pipeline Settings" button.
+10. In the left-hand side, under "Pipeline Settings", click on "General". Then, scroll down to the bottom of the page. In the "Pipeline Management" section, click on the white "Make Pipeline Public" button. If you are asked for confirmation, click on the green "Make Pipeline Public" button to confirm your decision.
+11. Go to https://buildkite.com/julialang/julia-release-1-dot-9 and click on the white "Pipeline Settings" button.
+12. In the left-hand side, under "Pipeline Settings", click on "Builds". Then, in the "Build Skipping" section, do the following actions:
+    - Step i: Make sure that the "Skip Intermediate Builds" checkbox IS checked.
+    - Step ii: Make sure that the "Cancel Intermediate Builds" checkbox IS checked.
+    - Step iii: Under "Skip Intermediate Builds", in the text box, enter the following text: `!master !main !release-*`
+    - Step iv: Under "Cancel Intermediate Builds", in the text box, enter the following text: `!master !main !release-*`
+    - Step v: Click on the green "Save Build Skipping" button.
+13. Go to https://buildkite.com/julialang/julia-release-1-dot-9 and click on the white "Pipeline Settings" button.
+14. In the left-hand side, under "Pipeline Settings", click on "GitHub". Then, in the "Branch Limiting" section, do the following actions:
+    - Step i: For "Branch Filter Pattern", enter the following text: `release-1.9 v1.9.*`
+    - Step ii: Click on the green "Save Branch Limiting" button.
+15. You are still on the "Pipeline Settings ⟶ GitHub" page. In the "GitHub Settings" section, do the following actions:
```

Comment:
I'm thinking for things like this, perhaps a picture is easier?

Reply:
Yeah.

I'm going to merge this just to get something in (currently we have nothing on main), but we should make follow-up PRs to add some screenshots.